src/components/pages/farmer-member/edit/index.tsx

```diff
@@ -1,5 +1,6 @@
-import { ArrowBackIcon, CheckIcon, EditIcon } from "@chakra-ui/icons";
-import { Accordion, Box, Button, Flex, Heading, Stack } from "@chakra-ui/react";
+/* eslint-disable @typescript-eslint/no-unused-vars */ //TODO: remove this after fixing map edit issue
```

Comment:
please remove this comment if not needed now.

Reply:
this is still needed.